data/dfndata/npc/agressiveplants.dfn

```diff
@@ -8,222 +8,235 @@
 {
+GETUO=reaper_lbr
+GETT2A=reaper_lbr
+GETUOR=reaper_lbr
+GETTD=reaper_lbr
+GETLBR=reaper_lbr
+GETAOS=reaper_aos
+GETSE=reaper_aos
+GETML=reaper_aos
+GETSA=reaper_aos
+GETHS=reaper_aos
+GETTOL=reaper_aos
+}
+
+[base_swamptentacle]
+{
+GET=base_agressiveplants
 NAME=#//a swamp tentacle
 ID=0x0042
-DIRECTION=RND
-BACKPACK
-GOLD=50 150
-PACKITEM=0x0f84,3
-STR=96 120
-DEX=66 85
-INT=16 30
-MANAMAX=0
-KARMA=-3000
-FAME=3000
-MAGICRESISTANCE=150 200
-PARRYING=550 650
-TACTICS=650 800
-WRESTLING=650 800
-DAMAGE=3 15
-DEF=15
 NPCWANDER=5
-NPCAI=2
-TOPROV=480
-TOPEACE=480 5
 RACE=18
-FLAG=NEUTRAL
 }
 
-[whippingvine]
+[swamptentacle]
+{
+GETUO=swamptentacle_lbr
+GETT2A=swamptentacle_lbr
+GETUOR=swamptentacle_lbr
+GETTD=swamptentacle_lbr
+GETLBR=swamptentacle_lbr
+GETAOS=swamptentacle_aos
+GETSE=swamptentacle_aos
+GETML=swamptentacle_aos
+GETSA=swamptentacle_aos
+GETHS=swamptentacle_aos
+GETTOL=swamptentacle_aos
+}
+
+[base_whippingvine]
 {
+GET=base_agressiveplants
 NAME=5000//a whipping vine
 ID=0x0008
-DIRECTION=RND
-BACKPACK
-PACKITEM=0x0f81,1 10
-PACKITEM=0x0f83
-LOOT=randomreagents,3
-STR=251 300
-DEX=76 100
-INT=26 40
-MANAMAX=0
-KARMA=-1000
-FAME=1000
-MAGICRESISTANCE=700
-PARRYING=500
-TACTICS=700
-WRESTLING=700
-DAMAGE=9 48
-DEF=10
 NPCWANDER=5
-NPCAI=2
-TOPROV=766
-TOPEACE=766 5
 RACE=18
-FLAG=NEUTRAL
-ORIGIN=aos
+ORIGIN=lbr
 }
 
-[quagmire]
+[whippingvine]
+{
+GETUO=whippingvine_lbr
+GETT2A=whippingvine_lbr
+GETUOR=whippingvine_lbr
+GETTD=whippingvine_lbr
+GETLBR=whippingvine_lbr
+GETAOS=whippingvine_aos
+GETSE=whippingvine_aos
+GETML=whippingvine_aos
+GETSA=whippingvine_aos
+GETHS=whippingvine_aos
+GETTOL=whippingvine_aos
+}
+
+[base_quagmire]
 {
 NAME=#//a quagmire
 ID=0x0315
-DIRECTION=RND
-BACKPACK
-GOLD=50 150
-STR=101 130
-DEX=66 85
-INT=31 55
-KARMA=-1500
-FAME=1500
-MAGICRESISTANCE=650 750
-PARRYING=500
-TACTICS=500 600
-WRESTLING=600 800
-DAMAGE=6 17
-DEF=20
-NPCWANDER=4
-FX1=-1
-FY1=-1
-FZ1=-1
-FX2=20
-NPCAI=2
-POISONSTRENGTH=3
-TOPROV=710
-TOPEACE=710 5
 RACE=18
-FLAG=NEUTRAL
 ORIGIN=aos
 }
 
-[bogthing]
+[quagmire]
+{
+GETUO=quagmire_lbr
+GETT2A=quagmire_lbr
+GETUOR=quagmire_lbr
+GETTD=quagmire_lbr
+GETLBR=quagmire_lbr
+GETAOS=quagmire_aos
+GETSE=quagmire_aos
+GETML=quagmire_aos
+GETSA=quagmire_aos
+GETHS=quagmire_aos
+GETTOL=quagmire_aos
+}
+
+[base_bogthing]
 {
 NAME=#//a bog thing
 ID=0x030c
-DIRECTION=RND
-BACKPACK
-GOLD=100 150
-LOOT=randomreagents,4
-PACKITEM=0x1be0,10
-STR=801 900
-DEX=46 65
-INT=36 50
-MANAMAX=0
-KARMA=-8000
-FAME=8000
-MAGICRESISTANCE=900 950
-PARRYING=500 650
-TACTICS=700 850
-WRESTLING=650 800
-DAMAGE=7 47
-DEF=20
-NPCWANDER=4
-FX1=-1
-FY1=-1
-FZ1=-1
-FX2=20
-NPCAI=2
-TOPROV=861
-TOPEACE=861 5
 RACE=18
-FLAG=NEUTRAL
 ORIGIN=aos
 }
 
-[bogling]
+[bogthing]
+{
+GETUO=bogthing_lbr
+GETT2A=bogthing_lbr
+GETUOR=bogthing_lbr
+GETTD=bogthing_lbr
+GETLBR=bogthing_lbr
+GETAOS=bogthing_aos
+GETSE=bogthing_aos
+GETML=bogthing_aos
+GETSA=bogthing_aos
+GETHS=bogthing_aos
+GETTOL=bogthing_aos
+}
+
+[base_bogling]
```

Comment:
This doesn't pull from base_plants so it loses DIRECTION, BACKPACK, NPCWANDER, and FX2.  The FLAG can be omitted if pulled from base_plants.